_includes/docs/extending/installing.md

```diff
@@ -0,0 +1,15 @@
+You can install extensions via the UI or the command line.
+
+### UI
```

Comment:
I think it is clarified, by stating in the downloading part, that only downloading via UI is discouraged.